L1Trigger/L1TTrackMatch/test/L1TrackObjectNtupleMaker_cfg.py

```diff
@@ -193,6 +194,16 @@
     process.pTkMHT = cms.Path(process.l1tTrackerHTMiss*process.l1tTrackerHTMissExtended)
     process.pTkMHTEmulator = cms.Path(process.l1tTrackerEmuHTMiss*process.l1tTrackerEmuHTMissExtended)
     process.pL1TrackTripletEmulator = cms.Path(process.l1tTrackTripletEmulation)
+    process.DispVert = cms.Path(process.DisplacedVertexProducer)
+    process.printTree = cms.EDAnalyzer("ParticleTreeDrawer",
```

Comment:
Can this be added as an optional part of the ObjectNtupleMaker rather than by default unless it is something that everyone would want running when creating the L1TTrack Objects

Reply:
I added a boolean in the cfg to run the displaced vertexing which is false by default here [d95ffe9](https://github.com/cms-l1t-offline/cmssw/pull/1257/commits/d95ffe97501c7e003b3bc9de7e83685ecc20a6a6)